### ISSUE TITLE
USA - August 10 1962 CTD

### ISSUE THREAD

**Issue by slagginwaggin** [TNODevs/TNO-Bug-Tracker#3139]
### Operating System

Windows 11

### HOI4 version

Bolivar, v1.14.4.d429

### TNO version

v1.5.0c

### Do you have other mods in use?

no

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

[USA_1962_08_01_07.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/14899342/USA_1962_08_01_07.zip)

### Describe the crash

On 10 August 1962, my game crashes to desktop. It is unavoidable in this save

### Steps to reproduce / Cause of the crash

Just go to 10 August 1962 on this save file

**Comment by Marok132** [comment]
Did this crash happen in each your new game? I should say it could be a save corruption. It can be fixed by starting a new game.

**Comment by Jack121Q** [comment]
Yes, sounds like a typical case of save file corruption to me. Sorry, nothing we can do.
Don't use cloud saves and have multiple auto-saves to prevent it.
